website/source/docs/agent/config_entries.html.md

```diff
@@ -21,53 +22,13 @@ Kind = "<supported kind>"
 Name = "<name of entry>"
 ```
 
-The two supported `Kind` configuration entries are detailed below.
+The supported `Kind` names for configuration entries are:
```

Comment:
It would be nice to have a bit of description for each of these so that users can understand the difference between them and determine which type they want. 

Reading the individual pages for the new kinds I'm having trouble figuring out the exact differences. It feels like they could use a table to differentiate between them and establish the order they are applied in.

Reply:
Added a new overview doc for now: `/docs/connect/discovery-chain.html`